playwright/src/main/java/com/microsoft/playwright/impl/ElementHandleImpl.java

```diff
@@ -304,13 +315,13 @@ public void scrollIntoViewIfNeeded(ScrollIntoViewIfNeededOptions options) {
 
   @Override
   public List<String> selectOption(String value, SelectOptionOptions options) {
-    String[] values = value == null ? null : new String[]{ value };
+    String[] values = value == null ? null : new String[]{value};
```

Comment:
Please revert code formatting changes here and in other places, they just clutter the PR.